include/ttmlir/Target/TTNN/program.fbs

```diff
@@ -18,9 +18,9 @@ table ToMemoryConfigOp {
 table ToLayoutOp {
   in: tt.target.TensorRef;
   layout: tt.target.TensorLayout;
-  dtype: tt.target.DataType;
+  dtype: tt.target.DataType = null;
```

Comment:
There's no need to set this to null, each field is optional in flatbuffers, and defaults to 0/null.

Reply:
For enums it has to be set, 0/null only works for flatbuffers::Offset types which tables get compiled to. That's why I previously had to add a NONE type to the enum type to make it behave similarly

Reply:
Oh I see, thanks for letting me know!